phoenix-core/src/test/java/org/apache/phoenix/execute/CorrelatePlanTest.java

```diff
@@ -172,17 +172,38 @@ public void testCorrelatePlanWithSingleValueOnly() throws SQLException {
                 {2, "2", "2", 20},
                 {5, "5", "5", 100},
         };
-        testCorrelatePlan(LEFT_RELATION, rightRelation, 1, 0, JoinType.Inner, expected);        
+        testCorrelatePlan(LEFT_RELATION, rightRelation, 1, 0, JoinType.Inner, expected);
     }
     
-    private void testCorrelatePlan(Object[][] leftRelation, Object[][] rightRelation, int leftCorrelColumn, int rightCorrelColumn, JoinType type, Object[][] expectedResult) throws SQLException {        
+    @Test
```

Comment:
Also, test for bind parameter for OFFSET, like this:

    SELECT * FROM T OFFSET ?

And check that the ResultSetMetaData and ParameterMetaData are ok.